.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,23 +1,53 @@
----
-name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
----
+This PR ...
 
-**Please describe the bug this fixes or the feature this adds.**
+## Changes
+-
+-
 
-**Please describe how you tested this change. Include unit tests whenever possible.**
+## Checklist
 
-**Did you create or modify any associated documentation with this change? If documentation is not included in PR, please link to related documentation.**
+### Testing
+- [ ] Tested locally?
+- [ ] Ran `quickTest all`?
+- [ ] Ran `./prime test` against local Docker ReportStream container?
+- [ ] Downloaded a file from `http://localhost:7071/api/download`?
+- [ ] Added tests?
 
-**If you added or modified HTML, did you check that it was 508 compliant?**
+### Specific Security-related subjects a reviewer should pay specific attention to
 
-**Please tag any specific reviewers you would like to review this PR**
+- Does this PR introduce new enpoints?
+    - new endpoint A
+    - new endpoint B
+- Does this PR include changes in authentication and/or authorization of existing endpoints?
+- Does this change introduce new dependencies that need vetting?
+- Does this change require changes to our infrastructure?
+- Does logging contain sensitive data?
+- Does this PR include or remove any sensitive information itself?
 
-**Please include the following checks for open source contributing?**
+If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:
 
-* [ ] Did you check for sensitive data, and remove any?
-* [ ] Are additional approvals needed for this change?
-* [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?
+- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
+    - **S**poofing (faking authenticity)
+        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
+    - **T**ampering (influence or sabotage the integrity of information, data, or system)
+    - **R**epudiation (the ability to dispute the origin or originator of an action)
+    - **I**nformation disclosure (data made available to entities who should not have it)
+    - **D**enial of service (make a resource unavailable)
+    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
```

Comment:
Including STRIDE is an awesome addition. Help gets the PR author thinking about attack surfaces.